http/vertx/src/test/java/io/quarkus/ts/http/minimum/OpenShiftHttpMinimumIT.java

```diff
@@ -0,0 +1,17 @@
+package io.quarkus.ts.http.minimum;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.OpenShiftScenario;
+import io.quarkus.test.services.QuarkusApplication;
+import io.restassured.specification.RequestSpecification;
+
+@OpenShiftScenario
+public class OpenShiftHttpMinimumIT extends AbstractIT {
+    @QuarkusApplication
+    static RestService app = new RestService();
```

Comment:
I think that, you can move this RestService to "AbstractIT"  and 

```
  @Override
    public RequestSpecification requests() {
        return app.given();
    }
```

could be removed. 

On AbstractIT just invoke restassurance. 

for example: 
```
public abstract class AbstractIT {

    @QuarkusApplication
    static RestService app = new RestService();
```


```
        given().get("/hello?name=you").then().statusCode(HttpStatus.SC_OK).body("content", is("Hello, you!"));

```

So, all platform classes should be empty in this case, because you don't have any particular configuration into any component.

Reply:
I would prefer to have all state in the endpoint classes and use abstract only as a source of common code.

Reply:
ok but I think this is common to all the scenarios

```
@QuarkusApplication
    static RestService app = new RestService();

```

Reply:
And this code Is not necessary:

```
@Override
    public RequestSpecification requests() {
        return app.given();
    }
```

so all commonCode would be in `vertxCommonsIT` (the abstract one)